ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -465,15 +465,14 @@ impl<'a> TraceBuilder<'a> {
             for op in safepoint.lives.iter() {
                 match op {
                     aot_ir::Operand::LocalVariable(iid) => {
-                        live_args.push(self.local_map[iid]);
+                        live_args.push(self.local_map[iid].clone());
```

Comment:
I realise I didn't need to make this change. I'm not sure we can even get constants here. Let me rectify this.